task4/controllers/authController.js

```diff
@@ -0,0 +1,37 @@
+const find = require('lodash/find');
+const jwt = require('jsonwebtoken');
+const users = require('../models/usersList');
+
+
+const matchCredentials = (name, password) => {
+  const user = find(users, { name });
+  if (user && user.password === password) {
+    return true;
+  }
+  return false;
+};
+
+const authUser = (req, res) => {
+  const { name, password } = req.body;
+  const isValidCredentials = matchCredentials(name, password);
+
+  if (isValidCredentials) {
+    const token = jwt.sign({ name }, 'privateKey');
```

Comment:
Export `privateKey` into config's file, please

Reply:
Okay, i will fix this issues